shared/src/main/scala/com/cibo/evilplot/demo/DemoPlots.scala

```diff
@@ -263,6 +212,88 @@ object DemoPlots {
       .render(plotAreaSize)
   }
 
+  lazy val pieChart: Drawable = {
+    val data = Seq("one" -> 1.5, "two" -> 3.5, "three" -> 2.0)
+    PieChart(data).rightLegend().render(plotAreaSize)
+  }
+
+  lazy val contourPlot: Drawable = {
+    import com.cibo.evilplot.plot._
+
+    import scala.util.Random
+
+    val data = Seq.fill(100)(Point(Random.nextDouble() * 20, Random.nextDouble() * 20))
+    ContourPlot(data)
+      .standard()
+      .xbounds(0, 20)
+      .ybounds(0, 20)
+      .render(plotAreaSize)
+  }
+
+  lazy val linePlot: Drawable = {
+    val data = (0 to 5)
+      .map(_.toDouble)
+      .zip(
+        Seq(
+          0.0, 0.1, 0.0, 0.1, 0.0, 0.1
+        ))
+      .map(Point.tupled)
+
+    LinePlot(
+      data
+    ).ybounds(0, .12)
+      .yAxis()
+      .xGrid()
+      .yGrid()
+      .frame()
+      .render(plotAreaSize)
+  }
+
+  lazy val heatmap: Drawable = {
```

Comment:
we should probably improve our heatmap example to use colors actually indicative of some density pattern

Reply:
It doesn't get displayed on the public facing page, its just in the resources and DemoPlots